deploy/provision.sh

```diff
@@ -1,23 +1,25 @@
 #!/bin/bash
 
 # Copyright (c) 2015 Intracom S.A. Telecom Solutions. All rights reserved.
-#
+# -----------------------------------------------------------------------------
 # This program and the accompanying materials are made available under the
 # terms of the Eclipse Public License v1.0 which accompanies this distribution,
 # and is available at http://www.eclipse.org/legal/epl-v10.html
 
 TEST_USER="jenkins"
```

Comment:
No it can be removed and can be created outside provision ie. Vagrantfile/Dockerfile. However we need this user for our purposes, for running our functional NSTAT tests.